modules/fundamental/src/advisory/endpoints/mod.rs

```diff
@@ -18,21 +20,27 @@ use trustify_common::{
     db::{query::Query, Database},
     decompress::decompress_async,
     id::Id,
-    model::{Paginated, PaginatedResults},
+    model::{BinaryData, Paginated, PaginatedResults},
 };
 use trustify_entity::labels::Labels;
 use trustify_module_ingestor::service::{Format, IngestorService};
 use trustify_module_storage::service::StorageBackend;
 use utoipa::{IntoParams, OpenApi};
 
-pub const CONTEXT_PATH: &str = "/api/v1/advisory";
+pub const CONTEXT_PATH: &str = "/v1/advisory";
 
-pub fn configure(config: &mut web::ServiceConfig, db: Database, upload_limit: usize) {
-    let advisory_service = AdvisoryService::new(db);
+pub fn configure(
+    config: &mut utoipa_actix_web::service_config::ServiceConfig,
+    db: Database,
+    upload_limit: usize,
+) {
+    let advisory_service = AdvisoryService::new(db.clone());
+    let purl_service = PurlService::new(db);
 
     config.service(
-        web::scope(CONTEXT_PATH)
+        utoipa_actix_web::scope(CONTEXT_PATH)
             .app_data(web::Data::new(advisory_service))
+            .app_data(web::Data::new(purl_service))
```

Comment:
I see that this is used in the `delete` fn, but if it wasn't there before, how did it work?

Reply:
It worked because we didn't scope the data. Now the instance is added to the scope only.